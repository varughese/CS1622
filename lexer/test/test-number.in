00012345678909 184467440737095516159999
123
-123
+123
0
9
924a
