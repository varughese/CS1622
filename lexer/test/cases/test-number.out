(1,NUM,"00012345678909")
(1,NUM,"184467440737095516159999")
(2,NUM,"123")
(3,SYM,"-")
(3,NUM,"123")
(4,SYM,"+")
(4,NUM,"123")
(5,NUM,"0")
(6,NUM,"9")
(7,NUM,"924")
(7,ID,"a")
